pkg/daemon/metrics.go

```diff
@@ -55,11 +55,11 @@ var (
 			Help: "state of kubelet health monitor",
 		})
 
-	// MCDRebootErr logs success/failure of reboot and errors
-	MCDRebootErr = prometheus.NewGaugeVec(
-		prometheus.GaugeOpts{
-			Name: "mcd_reboot_err",
-		}, []string{"node", "message", "err"})
+	// MCDRebootErr tallys failed reboot attempts
+	MCDRebootErr = prometheus.NewCounterVec(
```

Comment:
You don't need a CounterVec if you have no labels.

```suggestion
	MCDRebootErr = prometheus.NewCounter(
```